docker/develop/scripts/setup-openldap.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Dependencies for LDAP tests
+zypper install -y openldap2 openldap2-client hostname openssl-1_1
```

Comment:
That is a nice hack for now. Once we merge this PR, I will create a second one putting this into the image and removing this line then. Would that be okay for you?

Reply:
Ok